File: exercice3_2.py
import sys
import os

def main():
    if len(sys.argv) != 2:
        print(f"Usage : {sys.argv[0]} repertoire_R")
        return

    print(sys.argv[1])

    repertoire_courant = os.getenv("PWD")  # récupération du répertoire courant

    commande = f"cp -r {repertoire_courant} {sys.argv[1]}"

    os.system(commande)  # exécution de la commande

if __name__ == "__main__":
    main()

File: exercice3_2.c
#include<stdio.h>
#include<stdlib.h>
#include<unistd.h>

int main(int argc,char *arg[])
{

    if(argc != 2)
    {
        printf("Usage : %s repertoire_R\n",arg[0]);

        return (0);
    }
    char *repertoire_courant;
    char commande[50];

    printf("%s\n",arg[1]);
    repertoire_courant = getenv("PWD");//recuperation du repertoire courant
    sprintf(commande,"cp -r %s %s",repertoire_courant,arg[1]); //affectation du commande(cp -r ...) dans un variable nommé commande
    system(commande); //execution de la commande

    return (0);

}